pwa-shell/src/auth/keycloak/KeycloakProviderContainer.js

```diff
@@ -14,10 +14,13 @@ export const mapStateToProps = state => ({
   keycloak,
 });
 
-const parseUserInfo = ({ name, given_name, family_name }) => ({
-  fullname: name,
-  givenName: given_name,
-  familyName: family_name,
+const parseUserInfo = ({ name }) => ({
```

Comment:
where given name and family name will be fetched then?

Reply:
we don't actually use `given_name` and `family_name` around the app. Only `fullname` for side menu (LogoutContainer). Shall I place it back regardless on its usage?

Reply:
no, sorry, fullname is enough ATM. thanks :)